staker-rewards/test/rewards-calculator.test.js

```diff
@@ -241,25 +241,55 @@ describe("rewards calculator", async () => {
     assert.equal(rewards.totalRewards.isEqualTo(new BigNumber(0)), true)
   })
 
```

Comment:
There is a "should set total rewards to zero if pool is not authorized at start" test at line 224 which I think we need to delete.

Reply:
Nope, we didn't touch the `poolAuthorizedAtStart` requirement in this PR. We touched `poolDeauthorizedInInterval`

Reply:
Errr... wait. So say that I deauthorized sortition pool in this interval. I am getting rewards for this interval because I still need to run a node and I have my ETH locked. This interval ends and a new one starts. Am I getting rewards in the new interval?

Reply:
Nope,  because you will violate the `poolAuthorizedAtStart` requirement in the next interval. But in this PR we are discussing what happens if someone violates `poolDeauthorizedInInterval` during the current interval.

Reply:
Will there be another PR for issue #801? From the PR description, I assumed this is the only one closing the issue.

Reply:
Ok, I was sure we want to block rewards for deauthorized operators in the next intervals. Here is the change: https://github.com/keep-network/keep-ecdsa/pull/828/commits/b4bb8deae13d0c3d49ae6f571970fd956a5bac05

Reply:
Let's update README as well.